salesagility/SuiteCRM
# Fixed #4502 - Accounts Summary List not sorted - 7.9.7

## Description



Fixed accounts view summary not sorting

## Motivation and Context

Issue reference: #4502

## How To Test This

1. Enter accounts module.
2. Open history sub-panel.
3. Click on the "View Summary" popup.
4. Check that each tab is sorted correctly.

## Types of changes


### Final checklist
